readme.md

```diff
@@ -363,13 +363,15 @@ test(t => {
 });
 ```
 
-You can also use your own local Babel version:
+To use your own local Babel version, specify the version you want in your package.json, and ensure you're using npm@3 (or run `$ npm dedupe`), so the dependency will be deduped.
```

Comment:
I assume you're advocating deduping so that if the code-under-test requires `babel-core` it doesn't load two versions? If so, is that really worth mentioning?